JsonPointer/JsonPointer.csproj

```diff
@@ -33,6 +33,7 @@
 	<Nullable>enable</Nullable>
 	<NoWarn>IDE0290</NoWarn>
 	<PolySharpIncludeRuntimeSupportedAttributes>true</PolySharpIncludeRuntimeSupportedAttributes>
+    <IsAotCompatible Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net7.0'))">true</IsAotCompatible>	  
```

Comment:
If we put a public serializer context here, wouldn't that mean we can use that in other places, e.g. JsonSchema?

Reply:
Is this comment in the right place? I'm not sure which context you mean.

Reply:
We discussed in Slack.  Nevermind.